.github/ISSUE_TEMPLATE/update.md

```diff
@@ -3,48 +3,56 @@ name: Update
 about: Master issue for DevDB releases
 title: "{Version, i.e. 20Q4} UPDATE"
 labels: ''
-assignees: mgraber, SPTKL
+assignees:
 
 ---
 
 ## Update code
+
 - [ ] Add most recent full/half year to aggregate tables
 
 ## Update source data
 
 - [ ] Update version.env to:
-```
+
+```bash
 CAPTURE_DATE=2021-01-03
 CAPTURE_DATE_PREV=2020-06-30
 DOB_DATA_DATE=2020/01/01
 VERSION=20Q4
 VERSION_PREV=20Q2
```

Comment:
We should add the `VERSION_GEO` and a note about pinning the version so that aligns with the version of devdb we are building (i.e. 22Q4 should include the latest geosupport/Version_geo that would is released for that year)